inc/saiqueue.h

```diff
@@ -258,6 +271,32 @@ typedef enum _sai_queue_stat_t
 } sai_queue_stat_t;
 
 /**
+ * @brief Enum defining Queue deadlock event state.
+ */
+typedef enum _sai_queue_pfc_deadlock_event_type_t
+{
+    /** PFC deadlock detected */
+    SAI_QUEUE_PFC_DL_START = 0x00000001,
```

Comment:
The reason why I wanted to start with a non-zero value was to be able to distinguish if something was not initialized properly and just passed in as zero vs an actual configured value. 
I think this is a general issue in SAI all over and should be addressed. For now I'll set this to zero.